Include mission specific links

dchitti-stsci opened the pull request:
https://glitch.etc.stsci.edu/pandokia.cgi

cdsontag commented on pandokia/top_level.html:
```diff
@@ -16,12 +16,10 @@
-<li> <a href="CGINAME?query=day_report.2&test_run=pandeia_master_latest"><b>Latest Pandeia Report (AWS)</b></a>
+<li> <a href="CGINAME?query=day_report.2&test_run=pandeia_jwst_master_latest"><b>Latest JWST Pandeia Report</b></a>
+<li> <a href="CGINAME?query=day_report.2&test_run=pandeia_hst_master_latest"><b>Latest HST Pandeia Report</b></a>
+<li> <a href="CGINAME?query=day_report.2&test_run=pandeia_roman_master_latest"><b>Latest Roman Pandeia Report</b></a>
```
Love the forward thinking here.    It might be a hot minute before we have this running tho - does it make sense to leave this in but comment it out?

dchitti-stsci replied:
Commented out roman related html tags

cdsontag reviewed (approved):
Looks great.  Added a suggestion.  Can I please also ask that we add one more link to the top page for a place I am ALWAYS going to (but edit the url to get there)?  Call it "All pdk logs/runs" or something similar - a link to:  https://glitch.etc.stsci.edu

dchitti-stsci commented:
> Looks great. Added a suggestion. Can I please also ask that we add one more link to the top page for a place I am ALWAYS going to (but edit the url to get there)? Call it "All pdk logs/runs" or something similar - a link to: https://glitch.etc.stsci.edu
> 
> (I use this spot for both pyetc and pandeia)

Added a link for All PDK logs/runs -> https://glitch.etc.stsci.edu/pandokia.cgi

ariedel reviewed (approved):
Looks good to me!
